packages/wonder-blocks-dropdown/components/action-menu.js

```diff
@@ -241,7 +241,7 @@ export default class ActionMenu extends React.Component<Props, State> {
                               <ActionMenuOpenerCore
                                   {...eventState}
                                   disabled={disabled}
-                                  opened={opened}
+                                  opened={!!opened}
```

Comment:
`boolean | void` is no longer equivalent to `boolean`.

Reply:
Oh, that Flow change makes sense. No need to change anything here, just I think another way to fix this would be changing `opened` to required setting its default value to `false`.